### DISCUSSION TITLE
Release notes with one line description what was added or fixed (with link to specific issue)

### DISCUSSION THREAD

**Discussion by danieltomasz** [vslinko/obsidian-outliner#48] [Archive]
I wonder if adding a short description of what was done in the specific release in the futurevwould help with feature discoverability?
This could be done as a bullet list with a one-liner for every feature or fix with a possible link to the Github issue
(fixes might be skipped, only new features if they are present in the release might be described)

Great idea with using Discussions for talking about ideas :)

**Comment by vslinko** [comment]
Hi @danieltomasz ! Thank you for the suggestion!

Added some notes in last releases and will add notes in future ones.
